airflow/dag_processing/manager.py

```diff
@@ -348,951 +295,3 @@ def end(self):
         self._process.join(timeout=1.0)
         reap_process_group(self._process.pid, logger=self.log)
         self._parent_signal_conn.close()
-
```

Comment:
Merged this one into DagProcessorJobRunner - this way it is consistent with other Jobs - which "do something"  rather than create another object and "run" it.